app/src/main/java/org/simple/clinic/cvdrisk/CVDRisk.kt

```diff
@@ -0,0 +1,32 @@
+package org.simple.clinic.cvdrisk
+
+import android.os.Parcelable
+import androidx.room.Embedded
+import androidx.room.Entity
+import androidx.room.Index
+import androidx.room.PrimaryKey
+import kotlinx.parcelize.Parcelize
+import org.simple.clinic.patient.SyncStatus
+import org.simple.clinic.storage.Timestamps
+import java.util.UUID
+
+@Parcelize
+@Entity(
+    tableName = "CVDRisk",
+    indices = [
+      Index("patientUuid", unique = false)
+    ]
+)
+data class CVDRisk(
+    @PrimaryKey
+    val uuid: UUID,
+
+    val patientUuid: UUID,
+
+    val riskScore: String,
```

Comment:
Risk score is a numerical value, no?

Reply:
@msasikanth I think it should be string. As risk score will be a range when smoking status is unknown or when bmi is not known.